fcq/templates/fcq/fcq_display.html

```diff
@@ -0,0 +1,331 @@
+{% extends 'base.html' %}
+
+
+{% block content %}
+
+<br>
+<h1>Search for Professors</h1>
+<div id="error"></div>
+<div class="col-12 sticky-top" style="background-color: whitesmoke;">
```

Comment:
sticky-top. Nice. Did not know about this

Reply:
I've added this to the class search table header and search form and pushed changes